src/util/jira.ts

```diff
@@ -0,0 +1,26 @@
+import { JiraAuthor } from "../interfaces/jira";
+
+export const getJiraAuthor = (author?: Author): JiraAuthor => author ? {
+	avatar: author.avatar_url || author.avatarUrl || `https://github.com/${author.login}.png`,
```

Comment:
So author can either have avatar_url or avatarUrl parameters? What does it depend on?

Reply:
There were some places in the code that were referencing `avatarUrl` but I'm fairly sure that it's wrong as I'm fairly sure it's all `avatar_url` when it comes from the github data.  Didn't want to change it in case it might work.

Reply:
actually, the difference is between the REST API and the GraphQL API.  One uses `avatar_url` while the other does `avatarUrl` respectively.